participants/aaa/README.md

```diff
@@ -11,3 +11,7 @@ and this other part does something.  I forget why that I did it:
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
 If you can't use it, its kind of your problem, not mine!
+
+
+
+This is my line it is the best line I am a silly billy - Adam
```

Comment:
Readme files generally need something a bit more descriptive... how exactly are you a silly billy?